### PR TITLE
Heinrich Project Portofolio

### PR DESCRIPTION
**Pull request by HynRidge**

### CODE REVIEW

File: docs/team/hynridge.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+HelloFile is a file management application created as an extension to AddressBook - Level 3 (AB3),
+specifically made for tech savvy CS2103T CS students.
+By using HelloFile, students can tag frequently used files/folders with a short nickname, and open their files
+with intuitive commands.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Create the model of the app, by including `Tag`, `Label`, `FileAddress`, and `CurrentPath` class.
+(Pull request [\#81](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/81)) 
+  * What it does: allow developers to have a strong architecture to continue the app development.
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: I think this should probably be "allows"

File: docs/team/hynridge.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  * Justification: This feature improves the product significantly because these models are the key to the continuation of the app.
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: Maybe should be the "keys".

File: docs/team/hynridge.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  * Highlights: This enhancement affect the current AddressBook Level 3 at that point. So a lot of modification and refactoring were made on that period.
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: affects

File: docs/team/hynridge.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  * Credits: This enhancement was developed upon the AddressBook Level 3 model class, with some class being modified and some class being removed.
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: some classes

File: docs/team/hynridge.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+* **Project management**:
+  * Keep track of the milestone's deadline on each iteration.
+
+* **Enhancements to existing features**:
+  * Updated the HelpCommand to show details of specific command  (Pull requests [\#186](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/186))
+  * Updated the FindCommand to be case insensitive (Pull requests [\#207](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/207))
+  
+* **Documentation**:
+  * User Guide (UG):
+    * Added documentation for the feature `label` and `find` [\#169](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/169)
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: features

File: docs/team/hynridge.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    * Added `help COMMAND` in Features section [\#210](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/210)
+    * Added screenshots in the User Interface section. [\#212](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/212)
+    * Added some screenshots for the Basic Workflow section. [\#214](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/214)
+    
+  * Developer Guide(DG):
+    * Added description for feature `Tag` class and `Label` class [\#164](https://github.com/AY2021S1-CS2103T-F12-1/tp/pull/164)
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md]: features

**Review by Ma-Yueran — COMMENTED** [review]
Mostly looks good, but I think there are some small grammar mistakes.

**Review by luo-git — APPROVED** [review]
LGTM except for some minor things as pointed out by Yueran.

**Pull request merged**